MibTeX/src/de/mibtex/export/ExportHTML.java

```diff
@@ -214,6 +214,7 @@ public static String getHTMLTitle(BibtexEntry entry) {
 		if (entry.getPDFPath().exists()) {
 			htmlTitle += title;
 		} else {
+			System.out.println("Entry \"" + entry.getPDFPath() + "\" does not exist!");
```

Comment:
This is likely to be very annoying when using the command-line interface.

Reply:
I agree. Will comment this out since it is still nice to have for debugging.